packages/webapp/src/components/Form/Input/index.jsx

```diff
@@ -60,6 +61,17 @@ const Input = ({
 
   const onKeyDown = ['number', 'decimal'].includes(type) ? numberOnKeyDown : undefined;
 
+  useEffect(() => {
+    if (autoFocus) {
+      try {
+        input.current.focus();
```

Comment:
Even if the focus is on the input, you cannot start typing while the calendar is open, which I don't think is a good user experience.(the focus is on the calendar, and you can select a date using your keyboard) I would like to discuss the UX, so I will bring this up at standup.

Reply:
Never mind, it looks like that is what it is. I will approve!